GeoTIFF_Standard/standard/clause_4_terms_and_definitions.adoc

```diff
@@ -239,21 +216,29 @@ usually rectangular pattern of parallel scanning lines forming or corresponding
 
  [Source: ISO 19123:2005, 4.1.30]
 
+NOTE: In the the 1995 GeoTIFF v1.0, "A continuous planar space in which pixel values are
+visually realized."
+
 === rational <TIFF>
 
-a RATIONAL value is a fractional value represented by the ratio of two unsigned 4-byte integers
+a _rational_ value is a fractional value represented by the ratio of two unsigned 4-byte integers
 
 === rectified grid
+__georectified grid__
 
 grid for which there is an affine transformation between the grid coordinates and the coordinates of an external coordinate reference system
 
+Note 1 to entry:	If the coordinate reference system is related to the earth by a datum, the grid is a georectified grid.
+
+ [Source: ISO 19123:2005, 4.1.32]
 === referenceable grid
+__georeferenceable grid__
```

Comment:
referenceable grid / georeferenceable grid is not not mentionned elsewhere than in this section. To be deleted ?

```
$ grep -r referenceable .
./clause_4_terms_and_definitions.adoc:=== referenceable grid
```